src/Autofac.Integration.Wcf/AutofacInstanceContext.cs

```diff
@@ -25,7 +25,9 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.ServiceModel;
+using System.ServiceModel.Web;
```

Comment:
Why did this `using` become necessary?

Reply:
It didn't. That was probably added while testing various things and not cleaned up.